deploy/undeploy_all.sh

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+#todo: allow namespace option
```

Comment:
yes I think it only needs to add a namespace option so that the undeploy script knows where to delete the resources. I'll add the option and remove the comment.